src/sas/sascalc/calculator/geni.py

```diff
@@ -216,6 +216,12 @@ def _calc_Iqxy_magnetic(
 def orth(A, b): # A = 3 x n, and b_hat unit vector
     return A - np.outer(b, b)@A
 
+
+@njit
+def basis_vector(A, B): # A = 3 x n, and B = 3 x n
+    vec = np.cross(A, B)
+    return vec / np.sqrt(np.inner(vec, vec))   
```

Comment:
`np.inner` is not listed as a [supported numpy function](https://numba.readthedocs.io/en/stable/reference/numpysupported.html) in the current release of numba. This may interfere with JIT performance.

Reply:
It is changes now to np.dot, which should be supported.